src/tools/Support/ERCS_Logs/ERCS_AzureStackLogs.ps1

```diff
@@ -0,0 +1,1712 @@
+﻿<#
+.SYNOPSIS
```

Comment:
Thought: Would it make sense to convert it into psm1 and export the functions as cmdlets?